source/code/systems/resource_system/private/windows/resource_provider_dlls_win32.cxx

```diff
@@ -101,11 +102,12 @@ namespace ice
 
                 do
                 {
-                    ice::WString const file_name = file_data.cFileName;
+                    // TODO: Decide how to work with string static arrays.
```

Comment:
To count or not to count?

Reply:
For now this was an exception so we keep the constructor that sets the count to the array size encountered.